packages/neuron-ui/src/locales/zh.json

```diff
@@ -302,7 +302,9 @@
       "copy-balance": "复制余额",
       "copy-address": "复制地址",
       "create": "创建 {{name}} 资产账户",
-      "destroy": "销毁 {{name}} 资产账户"
+      "destroy": "销毁 {{name}} 资产账户",
+      "confirmation": "{{confirmationCount}} 个确认",
+      "confirmations": "{{confirmationCount}} 个确认"
```

Comment:
Is there any difference between these two lines?

Reply:
Different in English

Reply:
Please follow this suffix convention to adopt singular/plural expression

Ref: https://www.i18next.com/translation-function/plurals

Reply:
updated.